static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -2,83 +2,128 @@
 import fastdom from 'lib/fastdom-promise';
 import fetch from 'lib/fetch';
 
-const isCheckbox = element => element.type === 'checkbox';
+const targetUrl = 'https://callouts.code.dev-guardianapis.com';
```

Comment:
This will be better somewhere in `guardian.config`, which is an object generated in Scala [here](https://github.com/guardian/frontend/blob/master/common/app/templates/javaScriptConfig.scala.js). You can see there already is a `journalism` namespace, we can add it there.

Reply:
Okay sure so that is the same as this file? https://github.com/guardian/frontend/pull/21147/files#diff-3427c20bd152da9e20545325bd693877